src/Form/test-cases/index.js

```diff
@@ -239,5 +239,6 @@ export default [
     { html: 'googlecalendar_eventnotification.html' },
     { html: 'givesmart_login.html' },
     { html: 'vancity_login.html' },
-    { html: 'expensify.html' }
+    { html: 'expensify.html' },
+    { html: 'candidguidestar_checkout_signup.html' }
```

Comment:
Added the test case so that we won't regress on that specific site (and all similar sites).